### DISCUSSION TITLE
Mini Drawer

### DISCUSSION THREAD

**Discussion by vegulla001** [react-navigation/react-navigation#11377] [Ideas]
[
![Screenshot (181)](https://github.com/react-navigation/react-navigation/assets/74179166/f9fa64f4-6303-41c4-a40b-59ab3849f7aa)
](url)

**Comment by vegulla001** [comment]
Is it possible to add mini drawer in the latest drawer?

**Reply by billouboq** [reply]
You can pass a custom menu component to the drawer navigator so yes you can achieve that with react navigation drawer
